mobile/src/reducers/error/error.action.ts

```diff
@@ -0,0 +1,6 @@
+import { Action } from 'redux';
```

Comment:
Rename file to `error-load-failed.action.ts`

Reply:
there is no need, i think, do you want to have a separate file for each potential error action? I'd just rename it to errors/errors

Reply:
ok